app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+    has_many :votes
```

Comment:
If you want to destroy any votes for this work

```suggestion
    has_many :votes, dependent: :destroy
```